iojs.org/aliases.json

```diff
@@ -131,7 +131,7 @@
       "agiriabrahamjunior@gmail.com",
       "ahmadbamieh@gmail.com",
       "ammiller@linkedin.com",
-      "benjamin.zaslavsky@gmail.com",
+      "me@AhmadAwais.com",
```

Comment:
Nit: The email addresses are otherwise in alphabetical order, which is nice for lists that can tend to get long. Can you move this to be after Manil and before Michael to maintain alphabetical ordering?

Reply:
@amiller-gh are you going to be able to update?

Reply:
🙏 sorry, pseudo-vacation. Updated.